application/src/test/java/uk/gov/hmcts/net/ccd/definition/store/excel/SpreadSheetImportTest.java

```diff
@@ -343,26 +330,20 @@ public void importInvalidCaseTypeNoCConfigDefinitionFile() throws Exception {
         return hasColumn(is(key), is(value));
     }
 
-    private void assertBody(String contentAsString) throws IOException, URISyntaxException {
+    private void assertBody(String contentAsString) throws IOException, URISyntaxException, JSONException {
         assertBody(contentAsString, RESPONSE_JSON_V45);
     }
 
     private void assertBody(String contentAsString, String fileName)
-        throws IOException, URISyntaxException {
+        throws IOException, URISyntaxException, JSONException {
 
-        String expected = formatJsonString(readFileToString(new File(getClass().getClassLoader()
+        String expected = readFileToString(new File(getClass().getClassLoader()
             .getResource(fileName)
-            .toURI())));
+            .toURI()));
         expected = expected.replaceAll("#date",
             LocalDate.now().format(DateTimeFormatter.ofPattern("yyyy-MM-dd")));
 
-        assertEquals(removeGuids(expected), formatJsonString(removeGuids(contentAsString)));
-    }
-
-    private String formatJsonString(String string) throws IOException {
-        ObjectMapper objectMapper = new ObjectMapper();
-        return objectMapper.writerWithDefaultPrettyPrinter()
-            .writeValueAsString(objectMapper.readValue(string, Object.class));
+        JSONAssert.assertEquals(removeGuids(expected), removeGuids(contentAsString), false);
```

Comment:
Do you want to use `JSONCompareMode.LENIENT` rather than `false` here so that it is more immediately obvious as to the role of the 3rd paramter?